elide-standalone/README.md

```diff
@@ -33,7 +33,7 @@ To actually start your Elide application, add the following to your main method:
 
 ```java
 public class Main {
-  public static void main(String[] args) {
+  public static void main(String[] args) throws Exception {
```

Comment:
Rather than adding the unchecked exception to the signature, I would suggest adding it to the javadoc.

Same comment for the others.